db/migrate/20200321173123_create_rooms.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRooms < ActiveRecord::Migration[6.0]
+  def change
+    create_table :rooms do |t|
+      t.string :title
+      t.string :room_hash
```

Comment:
`t.string :room_hash, null: false, index: {unique: true}` so we don't need `add_index`